docs/architecture/microservices/microservice-ddd-cqrs-patterns/net-core-microservice-domain-model.md

```diff
@@ -26,7 +26,7 @@ An aggregate refers to a cluster of domain objects grouped together to match tra
 
 Transactional consistency means that an aggregate is guaranteed to be consistent and up to date at the end of a business action. For example, the order aggregate from the eShopOnContainers ordering microservice domain model is composed as shown in Figure 7-11.
 
-![A detailed view of the OrderAggregate folder: Address.cs is a value object, IOrderRepository is a repo interface, Order.cs is an aggregate root, OrderItem.cs is a child entity, and OrderStatus.cs is an enumeration class.](./media/image12.png)
+![Screenshot of the OrderAggregate folder and its classes.](./media/net-core-microservice-domain-model/vs-solution-explorer-order-aggregate.png)
```

Comment:
We need the complex image tag again, mainly because the text can't be read anywhere else:

```markdown
:::image type="complex" source="<folderPath>" alt-text="<alt-text>":::
<long description here>
:::image-end:::
```